.eslintignore

```diff
@@ -0,0 +1,4 @@
+.tmp/**
+app/bower_components/**
+app/scripts/vendor/**
+dist/**
```

Comment:
I don't know if any of these paths apply to this repo. We _may_ be able to nuke this file.